README.rst

```diff
@@ -43,6 +43,24 @@ Though conceptually names differ between cloud and infrastructure virtualization
 it was decided to stick to one representation in the codebase and interface to give a singlar API across
 all systems.
 
+Installation
+------------
+
+Wrapanapi can be installed via `pip` as `pip install wrapanapi`
+It is always a good idea to use virtualenv to install pip packages.
+
+For Linux Users, Depending on the distribution you are using, you may need to install following packages
+(or similar for your distrobution of linux):
+
+* libcurl-devel
+* openssl-devel
+* libxml2-devel
+* libxml2-static
+* gcc
+
+If you are in doubt if you really need these packages, you will hit errors during installation that will make it
```

Comment:
In lieu of this statement, perhaps point users towards the pypi packages that make these libraries required (looking at you `pycurl`). It may be helpful to also provide a link to http://pycurl.io/docs/latest/install.html :smile:

Reply:
Added a statement there.